File: Java/JavaApplication1/src/String/String2.java

package String;

public class String2 {
    
    public static void main(String[] args) {
        
        String s1= "JAAsia ANjum", s2= "jaasia anjum",s3;
        
        s3= s1.concat(s2);
        System.out.println("Concatenated : "+s3);
        
        System.out.println("Uppercase : "+s1.toUpperCase());
        System.out.println("Lowercase : "+s1.toLowerCase());
        
        boolean a= s1.startsWith("J");
        boolean b= s3.endsWith(s2);
        
        System.out.println("Starts with: "+a);
        System.out.println("Ends with: "+b);
        
        String[] names= {"jaas", "sia", "anjum"};
        
        for(String i: names)
        {
            System.out.println(i+" ");
        }
    }
}

File: Subin Sir/String2.c
#include<stdio.h>

int main()
{
    int i=0,length=0;
    char name[]= "Jaasia Anjum";

    while(name[i]!='\0')
    {
        length++;
        i++;
    }

    printf("String length : %d", length);
}
